tests/components/airzone/util.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for the Airzone integration."""
+
+from homeassistant.components.airzone import DOMAIN
+from homeassistant.components.airzone.const import SENSOR_TYPES
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, load_fixture
+from tests.test_util.aiohttp import mock_aiohttp_client
+
+CONFIG = {
+    CONF_HOST: "192.168.1.100",
+    CONF_PORT: 3000,
+}
+
+
+def airzone_requests_mock(mock):
+    """Mock requests performed to Airzone Local API."""
+
+    hvac_fixture = "airzone/hvac.json"
+
+    mock.post(
+        f"http://{CONFIG[CONF_HOST]}:{CONFIG[CONF_PORT]}/api/v1/hvac",
+        text=load_fixture(hvac_fixture),
+    )
+
+
+async def async_init_integration(
+    hass: HomeAssistant,
+):
+    """Set up the Airzone integration in Home Assistant."""
+
+    for description in SENSOR_TYPES:
+        description.entity_registry_enabled_default = True
+
+    with mock_aiohttp_client() as _m:
```

Comment:
https://github.com/home-assistant/core/pull/67666/files#r821451935

Reply:
Changed in 9e655b2693ded13aa0e4fa3de387da24e2c96cac